tests/lib/prepare.sh

```diff
@@ -775,6 +775,39 @@ uc20_build_corrupt_kernel_snap() {
     rm -rf "$REPACKED_DIR"
 }
 
+uc_write_bootstrap_wrapper() {
+    local SKELETON_PATH="$1"
+    local INJECT_ERR="${2:-false}"
+
+    cp -a /usr/lib/snapd/snap-bootstrap "$SKELETON_PATH"/usr/lib/snapd/snap-bootstrap.real
```

Comment:
I feel a lot of it should have been done by dropping a configuration for ExecStartPre and ExecStartPost. Maybe some clean up for the future.